### PR TITLE
fix: Added Whatsapp floating button, phone_number in issue and New Whatsapp message in Issue

### PR DESCRIPTION
**Pull request by mymi14s**
## Feature description
WhatsApp floating button added to website and v1 HTTP response utilized in web request.
The floating button directly open WhatsApp on mobile device to initiate conversation with the company AI driven chatbot. Twilio Number set as the WhatsApp button default.

Fixes cannot save Issue doctype because _assign_to no found in document.
Some document may bot be assigned to a user and there for cannot be updated, this pr fixed it by skipping the assign_to validation if none is found.

-- custom New WhatsApp button is added next to New Email in Issue footer
-- Reply Issue via whatsapp
-- Added reply to comment section below the form

image
image

### CODE REVIEW

File: one_fm/one_fm/custom/issue.json [hunk 1]
```diff
@@ -239,9 +239,138 @@
...
+   "read_only_depends_on": null,
+   "report_hide": 0,
+   "reqd": 0,
+   "search_index": 0,
+   "translatable": 1,
+   "unique": 0,
+   "width": null
+  }
+ ],
+ "custom_perms": [
```
Comment (pjamsheer) [inline one_fm/one_fm/custom/issue.json]: We dont need the custom perms.

File: one_fm/one_fm/custom/issue.json [hunk 2]
```diff
@@ -239,9 +239,138 @@
...
+   "report": 0,
+   "role": "Employee",
+   "select": 0,
+   "set_user_permissions": 0,
+   "share": 0,
+   "submit": 0,
+   "write": 1
   }
  ],
- "custom_perms": [],
```
Comment (pjamsheer) [inline one_fm/one_fm/custom/issue.json:244]: This is what we maintain in perms

File: one_fm/one_fm/custom/issue.json (hunk 1 above)
Comment (mymi14s) [reply one_fm/one_fm/custom/issue.json]: I would not know where the permission came from I may have to review it.

File: one_fm/one_fm/custom/issue.json (hunk 2 above)
Comment (mymi14s) [reply one_fm/one_fm/custom/issue.json:244]: @pjamsheer please review

Comment (pjamsheer) [reply one_fm/one_fm/custom/issue.json:244]: @mymi14s While export Customisation you can uncheck export permission
Screenshot 2022-03-27 at 2 05 11 PM

**Comment by mymi14s** [comment]
Custom permission resolved.

**Comment by mymi14s** [comment]
@pjamsheer Issue resolved.

**Review by pjamsheer — COMMENTED** [review]
LGTM @Ehtesham599 @mymi14s

**Review by Ehtesham599 — APPROVED** [review]

**Pull request merged**